src/test/java/edu/harvard/h2ms/web/init/H2msRestUtilsTest.java

```diff
@@ -48,16 +83,34 @@ public void format_month_success(Integer month, Integer year, String result){
     public static Object[][] getTestData_for_format_month_success_test() {
         return new Object[][]
                 {
-                        {2, 2018, "March (2018)"},
-                        {0, 2020, "January (2020)"},
-                        {11, 2018, "December (2018)"},
-                        {23, 2018, ""}
+                        // All valid values of months
+                        {0, 2020, "January (2020)"}, {1, 2019, "February (2019)"}, {2, 2018, "March (2018)"},
+                        {3, 2020, "April (2020)"}, {4, 2018, "May (2018)"}, {5, 2018, "June (2018)"},
+                        {6, 2020, "July (2020)"}, {7, 2018, "August (2018)"}, {8, 2018, "September (2018)"},
+                        {9, 2020, "October (2020)"}, {10, 2018, "November (2018)"}, {11, 2018, "December (2018)"},
                 };
     }
 
+    /**
+     * Testing Week Formatting from Utils with invalid values
+     */
+    @Test
+    public void test_format_month_invalid_values_result_in_null(){
+        H2msRestUtils  utils = new H2msRestUtils();
+        Assert.isTrue(utils.formatMonth(null,2018) == null);
```

Comment:
Could be cleaned up with Spring's [Assert.isNull method](https://docs.spring.io/autorepo/docs/spring/4.3.2.RELEASE/javadoc-api/org/springframework/util/Assert.html#isNull-java.lang.Object-). That said, it's a refactoring, don't feel forced to change it.